files/es/learn/common_questions/what_is_a_web_server/index.html

```diff
@@ -30,29 +30,29 @@ <h2 id="Sumario">Sumario</h2>
 
 <p><img alt="Basic representation of a client/server connection through HTTP" src="https://mdn.mozillademos.org/files/8659/web-server.svg" style="height: 200px; width: 600px;"></p>
 
-<p>Para publicar un sitio web, necesitarás un servidor web dinámico o estático.</p>
+<p>Para publicar un sitio web, necesitarás un servidor web estático o dinámico.</p>
 
-<p>Un <strong>servidor web estático</strong>, o pila, consiste en una computadora (hardware) con un servidor HTTP (software). Llamamos a este "estático" debido a que el servidor envía los archivos almacenados "tal cual" a tu navegador.</p>
+<p>Un <strong>servidor web estático</strong>, o pila, consiste en una computadora (hardware) con un servidor HTTP (software). Llamamos a "estático" a este servidor porque envía los archivos que aloja "tal como se encuentran" (sin modificarlos) a tu navegador.</p>
 
-<p>Un <strong>servidor web dinámico </strong>consiste en un servidor web estático con un software extra , lo común es que sea una <em>aplicación</em> servidor <em> y una </em> base de datos<em>. Llamamos a esto</em> "dinámico" por que la aplicacion servidor actualiza los archivos almacenados en la base de datos antes de enviarlos mediante el servidor HTTP.</p>
+<p>Un <strong>servidor web dinámico</strong> consiste en un servidor web estático con software adicional, habitualmente una <em>aplicación servidor</em> y una <em>base de datos</em>. Llamamos "dinámico" a este servidor porque la <em>aplicacion servidor</em> actualiza los archivos alojados, antes de enviar el contenido a tu navegador mediante el <em>servidor HTTP</em>.</p>
 
-<p>Por ejemplo, para ver la página que ves en tu navegador finalmente, el servidor aplicación puede mostrar el diseño HTML con contenido desde una base de datos. Sitios como MDN o Wikipedia tienen cientos de páginas web, que no son realmente archivos HTML, si no una estrucura HTML asociada a una gigantesca base de datos. Esto hace mas fácil y rápido el mantenimiento y entrega del contenido.</p>
+<p>Por ejemplo, para producir las páginas web que finalmente ves en tu navegador, la <em>aplicación servidor</em> podría llenar una plantilla HTML con contenido obtenido de una base de datos. Sitios como MDN o Wikipedia tienen miles de páginas web, que no son realmente archivos HTML. Típicamente, este tipo de sitios se componen de unas pocas plantillas HTML y una gigantesca base de datos, en vez de miles de documentos HTML estáticos. Esto hace mas fácil el mantenimiento y entrega del contenido.</p>
 
 <h2 id="Aprendizaje_activo">Aprendizaje activo</h2>
 
 <p><em>No hay aprendizaje activo disponible. <a href="https://developer.mozilla.org/en-US/docs/MDN/Getting_started">P</a><u>or favor, considere colaborar.</u></em></p>
```

Comment:
Un pequeño error de formateo
```suggestion
<p><em>No hay aprendizaje activo disponible. <a href="https://developer.mozilla.org/en-US/docs/MDN/Getting_started"></a><u>Por favor, considere colaborar.</u></em></p>
```